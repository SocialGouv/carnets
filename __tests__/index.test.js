import "@testing-library/jest-dom/extend-expect"

import { render } from "@testing-library/react"
import React from "react"

import Index from "../src/pages/index"

test("renders header index content", () => {
  const { asFragment } = render(<Index />)
  expect(asFragment()).toMatchSnapshot()
})
