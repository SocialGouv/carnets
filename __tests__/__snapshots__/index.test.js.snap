// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header index content 1`] = `
<DocumentFragment>
  <div
    class="content"
  >
    <ul
      class="teams"
    />
    <div
      class="page"
    >
      <div
        class="empty"
      >
        <p>
          Il n'y a aucune publication pour le moment.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
